box.json

```diff
@@ -16,7 +16,9 @@
     "description":"",
```

Comment:
Changes to this file should be reverted I think - has nothing to do with image derivatives :)